app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_like_counter
+
+  def update_like_counter
+    Post.find(post_id).update(likes_counter: Like.where(post_id).count)
+  end
```

Comment:
- [ ] Great job for making use of one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html), `after_save`. it is also good to make this method `private`. Kindly make it private. It should look something similar to this :point_down: 

```
  class Like < ApplicationRecord
  belongs_to :author, class_name: 'User'
  belongs_to :post

  after_save :update_like_counter

private  
def update_like_counter
    Post.find(post_id).update(likes_counter: Like.where(post_id).count)
  end
```